main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
```suggestion
while loop < 9:
```
This while loop is incorrectly formatted to have parentheses. This is JavaScript syntax and this should be in Python. Another thing is that it looks like this loop only runs a total of 8 times, but the project requires generating 10 stories. In this case, a for loop might be better since we know the exact number of times we need to loop.
```suggestion
while for i in range(10):
```